.github/workflows/prettier.yml

```diff
@@ -0,0 +1,39 @@
+name: Prettier
+
+# This action works with pull requests and pushes
+on:
+  - push
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+
+      # Make sure @sourcegraph/prettierrc is installed
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
Curious, what do you think about using the latest [lts](https://github.com/actions/setup-node/issues/26#issuecomment-871413720) version here?

Reply:
I updated it to read .nvmrc, then Renovate can upgrade it accordingly.